CONTRIBUTING.md

```diff
@@ -1,54 +1,87 @@
-# Guide to contributing
+# Eclipse Vert.x Community Contributing Guide
 
-Please read this if you intend to contribute to the project.
+This document describes a very simple process suitable for most projects in the Vert.x ecosystem. Projects are encouraged to adopt this whether they are hosted in the Vert.x Organization or not.
 
-## Legal stuff
+The goal of this document is to create a contribution process that:
 
-Apologies in advance for the extra work required here - this is necessary to comply with the Eclipse Foundation's
-strict IP policy.
+* Encourages new contributions.
+* Encourages contributors to remain involved.
+* Avoids unnecessary processes and bureaucracy whenever possible.
+* Creates a transparent decision making process which makes it clear how contributors can be involved in decision making.
 
-Please also read [this](http://wiki.eclipse.org/Development_Resources/Contributing_via_Git)
 
-In order for any contributions to be accepted you MUST do the following things.
+## Vocabulary
 
-* Sign the [Eclipse Foundation Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
-To sign the Eclipse CLA you need to:
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Committer** is a subset of contributors who have been given write access to the repository.
 
-  * Obtain an Eclipse Foundation userid. Anyone who currently uses Eclipse Bugzilla or Gerrit systems already has one of those.
-If you don’t, you need to [register](https://dev.eclipse.org/site_login/createaccount.php).
 
-  * Login into the [projects portal](https://projects.eclipse.org/), select “My Account”, and then the “Contributor License Agreement” tab.
+# Logging Issues
 
-* Add your github username in your Eclipse Foundation account settings. Log in it to Eclipse and go to account settings.
+Log an issue for any problem you might have. When in doubt, log an issue, any additional policies about what to include will be provided in the responses. The only exception is security dislosures which should be sent privately.
```

Comment:
Shouldn't we elaborate on "privately"? Giving a specific email address for example?